src/bundle/Resources/translations/custom_tags.en.xlf

```diff
@@ -18,8 +18,8 @@
             </trans-unit>
             <trans-unit id="8f9f4c953fc856ffdd89f2584789096bc29cd806"
                         resname="ezrichtext.custom_tags.ezyoutube.description">
-                <source>Add YouTube video</source>
-                <target>Add YouTube video</target>
+                <source>Add a YouTube video</source>
+                <target>Add a YouTube video</target>
```

Comment:
```suggestion
                <target>Add YouTube video</target>
```
To be consistent with omitting articles in other messages, except for `the` when referring to one or multiple item(s).